tutorials/fundamentals/installation.md

```diff
@@ -1,55 +1,101 @@
+
+```{note} 
+If you want to contribute back to the napari codebase, you should follow [this guide instead](https://napari.org/developers/contributing.html).
+```
 
-Welcome to the **napari** installation tutorial!
+## Prerequisites
 
-This tutorial will teach you how to do a clean install of **napari**. It is
-aimed at people that just want to use napari. For people also interested in
-contributing to napari please check our [contributing
-guidelines](https://github.com/napari/napari/blob/master/docs/developers/CONTRIBUTING.md)
-for more advanced installation procedures. At the end of the tutorial you should
-have napari successfully installed on your computer and be able to make the
-napari viewer appear.
+Prerequisites differ depending on how you want to install napari.
 
-## Installation
+### Installing as a Python package 
+This installation method allows you to use napari from Python to programmatically 
+interact with the app. It is the best way to install napari and make full use of
+all its features.
 
-### Which distribution to install
+It requires:
+- [Python 3.7+](https://www.python.org/downloads/)
+- the ability to install python packages via [pip](https://pypi.org/project/pip/) OR [conda-forge](https://conda-forge.org/docs/user/introduction.html)
 
-If you want to contribute back to the napari codebase, you should install from
-source code: see the [from source](#clone-the-repository-locally-and-install-in-editable-mode) section.
+You may also want:
+- an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
+[venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
-If you only wish to use napari as GUI app (and not from python code), the
-[bundled app](#installing-as-a-bundled-app)
-is the easiest way to install, it does not require having python pre-installed.
+### Installing the bundled app
+This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
+This installation method does not have any prerequisites. 
 
-If you are using napari from Python to programmatically interact with the app,
-you can install via pip, conda-forge, or from source.
+```{important}
+Note that the bundled app is still
+in active development, and may not be very stable. We strongly recommend 
+[installing as a Python package instead](#choose-which-distribution-to-install).
+```
 
-### From pip, with "batteries included"
+[Click here](#installing-as-a-bundled-app) to see instructions
+for installing the bundled app.
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
-3.7, 3.8, and 3.9 using pip:
-
-```sh
-pip install "napari[all]"
-```
+## Install as Python package
 
-*(See `Specifying a GUI Backend` below for an explanation of the `[all]`
-notation.)*
+Python package distributions of napari can be installed via `pip`, `conda-forge`, or from source.
 
 ````{important}
 While not strictly required, it is *highly* recommended to install
```

Comment:
```suggestion
While not strictly required, it is highly recommended to install
```

Personal preference, I'd prefer we didn't have such a mix of emphasis markers so close together. (There is bold text above for **(Highly recommended)**, the "Important" box is also for emphasis, I don't think we need a single word in italics to further emphasize things on top of all that)